### PR TITLE
Support modules in nodejs

### LINKED ISSUE #36: Package not working when using node
It seems that the bundle is not being exported in a way that it could be used with nodejs (with vanilla JS).
I found this pull request by @matextrem from the old/archived repo which fixes it: https://github.com/gnosis/cow-sdk/pull/14 but it was never merged into the main branch.

### PR DESCRIPTION
**Pull request by ramirotw**
Close #36 

Tested locally using https://github.com/ramirotw/cow-sdk-nodejs/tree/es6-import

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,4 +1,12 @@
...
+  "plugins": [
+    ["@babel/plugin-proposal-nullish-coalescing-operator"],
+    [
+      "babel-plugin-inline-import",
+      {
+        "extensions": [".json"]
+      }
+    ]
+  ],
+  "presets": [["@babel/preset-env", { "targets": { "node": "current" } }], "@babel/preset-typescript"]
```
Comment (ramirotw) [inline .babelrc:11]: I removed the `babel.config.js` file to unify all the babel config on this file

File: src/api/base/index.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { SupportedChainId } from '../../constants/chains'
 import { CowError } from '../../utils/common'
 import { Context } from '../../utils/context'
 import { Options } from '../cow/types'
+import fetch from 'cross-fetch'
```
Comment (ramirotw) [inline src/api/base/index.ts:6]: this was breaking nodejs usage

File: src/constants/index.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import contractNetworks from '@cowprotocol/contracts/networks.json'
 import { SupportedChainId as ChainId } from './chains'
 
-const { GPv2Settlement } = contractNetworks
+const { GPv2Settlement } = JSON.parse(contractNetworks as unknown as string) as typeof contractNetworks
```
Comment (ramirotw) [inline src/constants/index.ts:4]: this is a hackish solution until rollup (which microbundle uses) add support for import assertions https://github.com/rollup/rollup/issues/3799

having that supported we could do:

```js
import contractNetworks from '@cowprotocol/contracts/networks.json' assert { type: 'json' }
```

Comment (alfetopito) [reply src/constants/index.ts:4]: Wow, that's so hacking that's even funny 😆

File: package.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
   "module": "./dist/index.module.js",
   "exports": {
     "require": "./dist/index.js",
-    "default": "./dist/index.modern.js"
+    "default": "./dist/index.modern.mjs"
```
Comment (alfetopito) [inline package.json:10]: Is this because of the new microbundle version?

**Review by alfetopito — APPROVED** [review]
Awesome detective work, Ramiro!

File: package.json (hunk 4 above)
Comment (ramirotw) [reply package.json:10]: yes, https://github.com/developit/microbundle/pull/950

**Review by matextrem — APPROVED** [review]
Great Job! 🎉

File: src/api/base/index.ts (hunk 2 above)
Comment (W3stside) [reply src/api/base/index.ts:6]: thank you!

**Review by W3stside — APPROVED** [review]

**Pull request merged**